### ISSUE TITLE
Text: Link Default accessiblity

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#703]
**Issue:**
1. Hover state is only a color shift - we should not rely on just a color change to indicate that something is actionable.

2. Not seeing a pressed state, even when I'm using a mouse and clicking.

**Solution:**
1. Change hover state to make it more obvious something is actionable.

2. Incorporate pressed state.

**Comment by masoudmanson** [comment]
New designs needed!

**Comment by clarsen-czi** [comment]
Hey @masoudmanson, the hover and pressed states for `sdsStyle: default` (the blue one) should match the style and positioning of the underlines on the `sdsStyle: dashed` variant.

- `state: hover` the underline should be `Accent.border-hover`
- `state: pressed` the underline should be `Accent.border-pressed`
- `state: default` does not have an underline
